crawl-ref/source/makeitem.cc

```diff
@@ -647,6 +643,7 @@ bool is_missile_brand_ok(int type, int brand, bool strict)
     case SPMSL_EXPLODING:
         return type == MI_TOMAHAWK;
     case SPMSL_STEEL: // deliberate fall through
+        return false;
     case SPMSL_SILVER:
```

Comment:
There's one more thing. You can condense the `SPMSL_POISONED` and `SPMSL_STEEL` cases together and place them inside `#if TAG_MAJOR_VERSION == 34` markers:
```
#if TAG_MAJOR_VERSION == 34
    case ...:
    case ...:
        return false;
#endif
```
You should be able to use the same method of `git commit --amend` to update this.

Reply:
Something's wrong.
`$ git push --force
fatal: The current branch throwing has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin throwing


Administrator@ADMINIS-SDY MINGW64 /d/crawl (throwing)
$ git push --set-upstream origin throwing
remote: Permission to crawl/crawl.git denied to sdynet.
fatal: unable to access 'https://github.com/crawl/crawl.git/': The requested URL returned error: 403
`